pipeline/train/train.sh

```diff
@@ -57,15 +58,12 @@ for index in "${!datasets[@]}"; do
 
     if [ "${alignments}" != "None" ] ; then
       train_aln="${alns[index]}"
-      # alignments are currently used only for student that uses one stage of training
       echo "### Generating tsv dataset with alignments ${alignments}"
       paste <(${COMPRESSION_CMD} -dc "${train_set_prefix}.${src}.${ARTIFACT_EXT}") \
             <(${COMPRESSION_CMD} -dc "${train_set_prefix}.${trg}.${ARTIFACT_EXT}") \
             <(${COMPRESSION_CMD} -dc "${train_aln}") \
             >"${tsv_dataset}"
       rm "${train_aln}"
-      # when using tsv, marian requires --guided-alignments argument to be an index of the alignments in the tsv file
-      extra_params+=("--guided-alignment" "2")
```

Comment:
Though: I believe this is removed since they are now defined in `pipeline/train/configs/training/student.finetune.yml`

Reply:
yes